sdk/storage/Azure.Storage.Common/src/Shared/Constants.cs

```diff
@@ -551,6 +551,7 @@ internal static class GeoRedundantRead
         internal static class HttpStatusCode
         {
             internal const int NotFound = 404;
+            internal static readonly int[] s_retriableCodes = { 408, 429, 500, 502, 503, 504 };
```

Comment:
Nit - this name is really generic and I'd think you were using it for a retry policy.  Maybe this should be `s_DataLakeSetRecursiveAclUnhandledStatusCodes` or something crazy specific.